src/common/Common.h

```diff
@@ -669,7 +669,7 @@ namespace Sapphire::Common
   {
     Common::ActionEffectType effectType;
     Common::ActionHitSeverityType hitSeverity;
-    uint8_t unk;
+    Common::ActionHitSeverityType healSeverity;
```

Comment:
this isn't right in general, `hitSeverity`, `healSeverity` and `param` are basically like param0/1/2 and depends on what `effectType` is set to

Reply:
yea I'll rename them